src/thor/isochrone.cc

```diff
@@ -322,8 +322,10 @@ ExpansionRecommendation Isochrone::ShouldExpand(baldr::GraphReader& /*graphreade
   // max but need to consider others so we just continue here. Tells MMExpand function to skip
   // updating or pushing the label back
   // prune the edge if its start is above max contour
-  if (time > max_seconds_ && dist > max_meters_)
-    return ExpansionRecommendation::prune_expansion;
+
+  ExpansionRecommendation recommendation = time > max_seconds_ && dist > max_meters_
```

Comment:
nit: can you put parantheses around the check? it's not breaking anything, it's mostly consistent style I guess:)